StortSpelProjekt/Engine/src/Renderer/HLSL/LightCalculations.hlsl

```diff
@@ -93,7 +93,7 @@ float3 CalcDirLight(
 		shadow = CalculateShadow(fragPosLightSpace, dirLight.textureShadowMap);
 	}
 
-	DirLightContribution = (kD * albedo / PI + specular) * radiance * NdotL;
+	DirLightContribution = (kD * albedo / PI + specular) * radiance * NdotL * dirLight.baseLight.intensity;
```

Comment:
Would like radiance normalized since intensity now scales the color up. (2.0, 0, 1.0) should be == (1.0, 0, 0.5)

Reply:
(Does not have to be normalized in the shader)

Reply:
now normalised